armory/__main__.py

```diff
@@ -53,11 +53,12 @@ def __call__(self, parser, namespace, values, option_string=None):
         elif values.lower() in DOCKER_IMAGES:
             setattr(namespace, self.dest, DOCKER_IMAGES[values])
         else:
-            raise argparse.ArgumentError(
-                self,
-                f"{values} invalid.\n"
-                f" must be one of {DOCKER_IMAGES} or {images.ALL}",
+            print(
+                f"WARNING: {values} not in "
+                f"{list(DOCKER_IMAGES.keys()) + list(DOCKER_IMAGES.values())}. "
+                "Attempting to load custom Docker image."
             )
+            setattr(namespace, self.dest, values)
```

Comment:
Can you add documentation to how this command would look? Perhaps in the central README:
https://github.com/twosixlabs/armory#usage

But if not maybe somewhere else in the docs.